src/qt/sendcoinsdialog.h

```diff
@@ -98,6 +99,7 @@ private Q_SLOTS:
     void coinControlClipboardLowOutput();
     void coinControlClipboardChange();
     void updateFeeSectionControls();
+    void updateNumberOfBlocks(int count, const QDateTime& blockDate, double nVerificationProgress, bool headers, SynchronizationState sync_state);
```

Comment:
nit: `updateNumberOfBlocks` slot name for an object that knows nothing about "number of blocks" looks a bit weird. But I cannot suggest a better name (